cli/cmd/main.go

```diff
@@ -185,6 +207,92 @@ func main() {
 					},
 				},
 			},
+			// TODO: Add help stuff.
+			{
+				Name:  "data",
+				Usage: "download synced data",
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:     dataFlagDestination,
+						Required: true,
+						Usage:    "output directory for downloaded data",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagType,
+						Required: true,
+						Usage:    "data type to be downloaded: either binary or tabular",
+					},
+					&cli.StringSliceFlag{
+						Name:     dataFlagOrgs,
+						Required: false,
+						Usage:    "org filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagLocation,
+						Required: false,
+						Usage:    "location filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagRobotID,
+						Required: false,
+						Usage:    "robot_id filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagPartID,
+						Required: false,
+						Usage:    "part_id filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagRobotName,
+						Required: false,
+						Usage:    "robot_name filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagPartName,
+						Required: false,
+						Usage:    "part_name filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagComponentType,
+						Required: false,
+						Usage:    "component_type filter",
+					},
+					&cli.StringFlag{
+						Name:     dataFlagComponentModel,
+						Required: false,
+						Usage:    "component model filter",
```

Comment:
supernit: for consistency, `component_model` and `component_name` below

Reply:
Done